apps/web/src/pages/templates/workflow/digest/WillBeSentHeader.tsx

```diff
@@ -51,125 +19,30 @@ const Highlight = ({ children }) => {
     </b>
   );
 };
-export const WillBeSentHeader = ({ index }) => {
-  const { watch, setValue } = useFormContext();
-
-  const type = watch(`steps.${index}.metadata.type`);
-  const unit = watch(`steps.${index}.metadata.unit`);
-  const backoff = watch(`steps.${index}.metadata.backoff`);
-  const atTime = watch(`steps.${index}.metadata.timed.atTime`);
-  const weekDays = watch(`steps.${index}.metadata.timed.weekDays`) || [];
-  const monthDays = watch(`steps.${index}.metadata.timed.monthDays`) || [];
-  const amount = watch(`steps.${index}.metadata.amount`);
 
-  useEffect(() => {
-    if (!backoff) {
-      return;
-    }
-    if (type !== DigestTypeEnum.TIMED) {
-      return;
-    }
-    setValue(`steps.${index}.metadata.backoff`, false);
-  }, [backoff, type, index]);
+export const WillBeSentHeader = ({ index }: { index: number }) => {
```

Comment:
"Will be sent" field accordion header, there are a lot of different logic to show different labels depending on what user has selected